uber/configspec.ini

```diff
@@ -7,6 +7,10 @@
 # after a deadline or on-site.
 dev_box = boolean(default=True)
 
+# Redirect 404s to Uber's default URL
+default_url = string(default="/uber/common/")
+default_url_priority = integer(default=1)
```

Comment:
Our nginx config will now pass requests for "/profiler" through to uber. We're setting the `default_url` so those requests will just forward to "/uber/common".